src/components/form/examples/basic-form.tsx

```diff
@@ -1,11 +1,11 @@
 import { ValidationStatus } from '@limetech/lime-elements';
```

Comment:
> > > By the way. I'm not sure what will happen to fields such as chipset, which can grow in height. We will probably need to make tweaks in them (?). thinking
> > 
> > 
> > Very good point!
> 
> This is not limited to chip sets. As soon as a custom component is used (which a chip-set will be if it's used), anything can happen to the layout since the components can grow/shrink however they see fit

I've tested this with pickers (I hacked the custom-component a little, see the two tmp-commits I've added), and the resizing doesn't create any issues with getting cut off or overflowing above or beneath other components. The height of the row the item is in is simply increased, so there are holes created around it (unless it's alone in its row of course), but that's not breaking any functionality.

I'm placing this comment on a line in the diff so we can thread any future comments, and I'll mark the previous comments quoted here as resolved, to make it easier to see which comments are still left to resolve.

Reply:
I've now added a few textareas as well, and tested around a bit, and it appears that a component is free to grow so long as the rowSpan has not been defined for that component.

As soon as `rowSpan` is set, even if set to `1`, the component's height cannot be changed.

I'm pushing another tmp-commit with a few more textareas and another picker.

Reply:
@adrianschmidt thanks. It sounds great!

This sounds like a good addition to the documentations:
> a component is free to grow so long as the rowSpan has not been defined for that component. As soon as `rowSpan` is set, even if set to `1`, the component's height cannot be changed.

Reply:
Good point! 👍

Reply:
⚡

Can you take a look @Kiarokh? (It's in the latest fixup at the time of writing.)

I added this note:

```diff
+ * :::note
+ * If you do *not* set the `rowSpan` for a component, it can stretch vertically
+ * within its row, and the row will simply expand with the component.
+ *
+ * If you *do* set a `rowSpan`, even if you set it to `1`, the component is
+ * fixed to that height. What happens to any potential overflow depends on the
+ * component.
+ * :::
```

But with that, I didn't think the passage about always assigning at least two columns to textareas made as much sense anymore, so I changed it like this:

```diff
  * it does not really make sense to stretch them vertically, and we strongly
  * recommend you not to!
  *
- * But there are some exceptions. One of them is `limel-input-field` with `type='textarea'`.
- * It's always recommended to assign at least two rows for a textarea, to improve
- * its usability. Since this is a component that is supposed to show multi-lined
- * pieces of text.
+ * But there are some exceptions. One of them is `limel-input-field` with
+ * `type='textarea'`.
  *
  * Also, if you create a custom component for your form—let's say a map—you
  * can use `rowSpan` to increase the height of your custom component.
```

Reply:
You're right! The next section is better to be removed. But I'm just thinking if now somehow magically due to latest changes, whether applying an inline style for `min-height` works better than `height`? This way, rows won't be "locked".

First time we tested, `min-height` wouldn't help the `textarea` to visually and vertically expand inside its container grid cell. But maybe now it does.

Reply:
Also something off-topic, but I must say, I really dislike the way this `–` character is supposed to be used when I think about it... 😞 

> [···] form—let's say a map—you [···]

It has serious usability problems in my opinion 😄 I'd read those tow words before and after that character as attached words. 😢 
the difference between this: `-` (normal dash) and this: `–` longer dash is simply too little to notice for normal people (like myself) - –. Is there only one standard for writing this?

Reply:
Do you mean this fix?
![image](https://user-images.githubusercontent.com/154725/101331159-de7c1380-3873-11eb-8458-e0b74277c902.png)

Reply:
yes exactly

Reply:
Way ahead of you 😉